config/client_config/dc/system/config/templates/features/enroll_app/enroll_app.yml

```diff
@@ -308,6 +308,8 @@ registry:
             item: 85
           - key: :child_age_limit
             item: 20
+          - key: :default_password_length_range
+            item: 8..20
```

Comment:
why we don't change this to DC to be 12..20 and leave ME config as 8..20, that way we don't need the other Feature flag STRONG_PASSWORD_LENGTH_IS_ENABLED